tests/test_objective_funs.py

```diff
@@ -206,7 +206,7 @@ def test(eq):
             f = obj.compute_unscaled(*obj.xs(eq))
             f_scaled = obj.compute_scaled_error(*obj.xs(eq))
             np.testing.assert_allclose(f, 1.3 / 0.7, rtol=5e-3)
-            np.testing.assert_allclose(f_scaled, 2 * (1.3 / 0.7), rtol=5e-3)
+            np.testing.assert_allclose(f_scaled, 2 * (1.3 / 0.7), rtol=8e-3)
```

Comment:
Is this loss of accuracy acceptable?

Reply:
yes because the convergence should be exponential. if half nodes stays at same order of accuracy we're good